app/src/cards/MapCard.tsx

```diff
@@ -34,46 +33,65 @@ function MapCard(props: {
 
   // TODO - make sure the legends are all the same
   // TODO - pull these from the data itself
-  const RACES = [
-    "Total",
-    "American Indian and Alaska Native alone",
-    "American Indian and Alaska Native alone (Non-Hispanic)",
-    "Asian alone",
-    "Asian alone (Non-Hispanic)",
-    "Black or African American alone",
-    "Black or African American alone (Non-Hispanic)",
-    "Hispanic or Latino",
-    "Native Hawaiian and Other Pacific Islander alone",
-    "Native Hawaiian and Other Pacific Islander alone (Non-Hispanic)",
-    "Some other race alone",
-    "Some other race alone (Non-Hispanic)",
-    "Two or more races",
-    "Two or more races (Non-Hispanic)",
-    "White alone",
-    "White alone (Non-Hispanic)",
-  ];
+  const RACES = props.nonstandardizedRace
+    ? [
+        "Total",
+        "American Indian and Alaska Native alone",
+        "American Indian and Alaska Native alone (Non-Hispanic)",
+        "Asian alone",
+        "Asian alone (Non-Hispanic)",
+        "Black or African American alone",
+        "Black or African American alone (Non-Hispanic)",
+        "Hispanic or Latino",
+        "Native Hawaiian and Other Pacific Islander alone",
+        "Native Hawaiian and Other Pacific Islander alone (Non-Hispanic)",
+        "Some other race alone",
+        "Some other race alone (Non-Hispanic)",
+        "Two or more races",
+        "Two or more races (Non-Hispanic)",
+        "White alone",
+        "White alone (Non-Hispanic)",
+      ]
+    : [
+        "American Indian/Alaskan Native, Non-Hispanic",
+        "Asian, Non-Hispanic",
+        "Black, Non-Hispanic",
+        "Hispanic",
+        "Other race, Non-Hispanic",
+        "White, Non-Hispanic",
+      ];
+
   const [race, setRace] = useState<string>(RACES[0]);
+  useEffect(() => {
+    setRace(RACES[0]);
+  }, [RACES, props.metricConfig]);
```

Comment:
Same thing here about updating dependent state happening in a subsequent react update

Reply:
I'm hoping this useEffect will go away once we fix race standardization.... If not we can address it then. I'll make a TODO to emphasize this should go away